tools/pfe-tools/package.json

```diff
@@ -41,12 +41,12 @@
     "url": "https://github.com/patternfly/patternfly-elements/issues"
   },
   "scripts": {
-    "build": "tsc -b .",
+    "build": "node _scripts/build.js",
     "clean": "git clean -fx -e node_modules -e tsconfig.tsbuildinfo .",
-    "dev": "node _scripts/dev.js",
+    "dev": "node _scripts/build.js -- watch",
     "prepublishOnly": "run-s prepublishOnly:*",
     "prepublishOnly:analyze": "npm run analyze",
-    "prepublishOnly:build": "tsc -b ."
+    "prepublishOnly:build": "node _scripts/build.js"
```

Comment:
can you run `prepublish` then `npm pack` and confirm the contents of the tarballs?

Reply:
Also, this won't generate the .d.ts files, which we'd like in the tarball

Reply:
It's complaining about the `npm run analyze` script not being available.
![Screen Shot 2021-12-14 at 8 40 00 AM](https://user-images.githubusercontent.com/3428964/146009223-ddaee13d-96dd-4b79-91e4-00dd9649af85.png)